frontend/ui/widget/keyboardlayoutdialog.lua

```diff
@@ -27,14 +27,11 @@ local KeyboardLayoutDialog = InputContainer:new{
     modal = true,
     width = Screen:getWidth() * 0.8,
     face = Font:getFace("cfont", 22),
-
```

Comment:
Ftr, while not exactly applicable here there's a difference between spacing for ordering and random stray lines (like for the closing `}`). :-)

Reply:
i got rid of them it was mostly there as i copied from input dialog